app/src/components/Cards/years-selection-card.tsx

```diff
@@ -0,0 +1,181 @@
+"use client";
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Card,
+  Heading,
+  Text,
+} from "@chakra-ui/react";
+
+import React, { useState } from "react";
+import { format } from "date-fns";
+import { de, enUS, es, fr, it, pt } from "date-fns/locale";
+import { api } from "@/services/api";
+import { useRouter } from "next/navigation";
+
+const localesMap: Record<string, any> = {
+  en: enUS,
+  es: es,
+  pt: pt,
+  fr: fr,
+  de: de,
+  it: it,
+};
+
+function YearCard({
+  isActive,
+  year,
+  inventoryId,
+  cityId,
+  lastUpdate,
+  lng,
+}: {
+  isActive: boolean;
+  cityId: string;
+  year: number;
+  lastUpdate: Date;
+  inventoryId: string;
+  lng: string;
+}) {
+  const [setUserInfo] = api.useSetUserInfoMutation();
+  const router = useRouter();
+
+  const onClick = () => {
+    setUserInfo({ defaultInventoryId: inventoryId, cityId: cityId });
+    router.push(`/${inventoryId}`);
+  };
+
+  return (
+    <Card
+      onClick={onClick}
+      key={year}
+      className="flex flex-row h-[120px] duration-300 cursor-pointer items-center px-4 gap-4 shadow-none"
+      backgroundColor={isActive ? "background.neutral" : ""}
+      borderWidth="2px"
+      border="solid"
+      borderColor={isActive ? "interactive.secondary" : "border.neutral"}
+      _hover={
+        !isActive
+          ? {
+              backgroundColor: "background.neutral",
+            }
+          : {}
+      }
+    >
+      <Box className="flex flex-col gap-[8px]">
+        <Heading
+          fontSize="title.sm"
+          fontWeight="medium"
+          lineHeight="20"
+          letterSpacing="wide"
+          color="content.primary"
+        >
+          {year}
+        </Heading>
+        <Text
+          fontWeight="regular"
+          color="interactive.control"
+          lineHeight="20"
+          letterSpacing="wide"
+        >
+          Last Update:{" "}
+          {format(new Date(lastUpdate), "dd/MM/yyyy", {
```

Comment:
Let's make sure that we use date formats from the currently selected language. Should be available in i18next.

Reply:
will create a follow up tickets